README.md

```diff
@@ -2,34 +2,81 @@
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/d89a0d60-217a-4563-ad03-65ccc90b9222/deploy-status)](https://app.netlify.com/sites/nubook/deploys)
 
-NUbook is the handbook and high-level documentation for the NUbots team. You can read the latest version at https://nubook.netlify.com/.
+NUbook is the handbook and high-level documentation for the NUbots team. You can read the latest version at <https://nubook.netlify.com/.>
 
 Read on if you want to add or update content.
 
 ## Getting started
 
-You'll need [Git](https://git-scm.com/), [Node.js](https://nodejs.org/en/) and [Yarn](https://yarnpkg.com/en/) installed. Get those first, then:
+### Install prerequisites
 
-1. Clone this repo and `cd` into the cloned directory
+You'll need [Git](https://git-scm.com/), [Node.js](https://nodejs.org/en/) and [Yarn](https://classic.yarnpkg.com/en/) installed.
+
+<details>
+<summary>Ubuntu installation instructions</summary>
+
+1. Install Git by running the following commands:
+
+```
+sudo apt update
+sudo apt install git
+```
+
+2. Install the latest version of Node.js by following the instructions [here](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions) to (pick the latest version).
```

Comment:
```suggestion
2. Install the latest version of Node.js by following the instructions [here](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions).
```

Reply:
😄 I probably meant to say pick the latest LTS version.